bitnami/kafka/3.2/debian-11/rootfs/opt/bitnami/scripts/libkafka.sh

```diff
@@ -772,8 +772,8 @@ kafka_zookeeper_configure_tls() {
     elif [[ "$KAFKA_ZOOKEEPER_TLS_TYPE" = "PEM" ]] && [[ -f "$KAFKA_CERTS_DIR"/zookeeper.keystore.pem ]] && [[ -f "$KAFKA_CERTS_DIR"/zookeeper.keystore.key ]]; then
         # Concatenating private key into public certificate file
         # This is needed to load keystore from location using PEM
-        cat "$KAFKA_CERTS_DIR"/zookeeper.keystore.key >>"$KAFKA_CERTS_DIR"/zookeeper.keystore.pem
-        keystore_location="${KAFKA_CERTS_DIR}/zookeeper.keystore.pem"
+        keystore_location="${KAFKA_CERTS_DIR}/zookeeper.keypair.pem"
+        cat "${KAFKA_CERTS_DIR}/zookeeper.keystore.{pem,key}" > "$keystore_location"
```

Comment:
Unfortunately, [brace expansion](https://www.gnu.org/software/bash/manual/html_node/Brace-Expansion.html) will not work in a quoted string.

In the repository, we tend to favor the style of variable interpolation with curly braces and quoting the whole string, for example `"${var}/foo/bar"`. Although you can find exceptions. In this case, following this style convention would prevent brace expansion from working. So we need:

1)
```shell
cat "$KAFKA_CERTS_DIR"/zookeeper.keystore.{pem,key} > "$keystore_location"
```

or

2)
```shell
cat "${KAFKA_CERTS_DIR}/zookeeper.keystore.pem" "${KAFKA_CERTS_DIR}/zookeeper.keystore.key" > "$keystore_location"
```

I would go with the second option to follow the common style convention, but option one will also work.

Reply:
Yes, this should fix the current implementation. Apologies for not checking for the common style. I'll stick with the second one as well.

Reply:
No problem at all!! Thank you for adding the suggestion.